### PR TITLE
fix: CKV_AWS_91 checkov alert

### PR DESCRIPTION
**Pull request by ndegwajohn**
# Description
This PR fixes `CKV_AWS_91 : Ensure the ELBv2 (Application/Network) has access logging enabled` checkov alert.
It also adds new github workflow files.

## Features/Fixes/Patches/Changes list:

Please check the [CHANGELOG.md](https://github.com/boldlink/<REPOSITORY>/blob/fix/CKV_AWS_91/CHANGELOG.md#116---2023-02-03)

### PR Owners Checklist:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you checked there aren't open [Pull Requests](../../../pulls) of upstream or parallel code that will cause conflicts?
* [x] Have you updated the `CHANGELOG.md`?
* [x] Have you updated the `README.md`(s)?
* [x] OWNER: Does your submission pass?
    * [ ] Pre-commit (attach logs/print screen to this PR)
    * [ ] Checkov/terrascan (attach logs/print screen to this PR)
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### PR Reviewers Checklists?
* [ ] Are all your comments/changes resolved?
* [ ] Are you happy with the OWNER checklists and additional information provided?
* [ ] Does your review tests pass?
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -1,7 +1,6 @@
 locals {
   create_task_definition = var.deployment_controller_type != "EXTERNAL" && var.create_task_definition ? true : false
   create_lb_sg           = var.create_load_balancer && length(var.lb_security_group_ingress) > 0 ? true : false
-  legible_lb_type        = title(var.load_balancer_type) == "Application" || title(var.load_balancer_type) == "Network" ? true : false
```
Comment (mariomerco) [inline locals.tf:4]: Is the `? true : false` really necessary here? wouldn't `title(var.load_balancer_type) == "Application" || title(var.load_balancer_type) == "Network"` by itself be a boolean already? not sure if Terraform supports this, though.

**Review by mariomerco — COMMENTED** [review]
Please take a look at my comments in the boolean part

File: locals.tf (hunk 1 above)
Comment (ndegwajohn) [reply locals.tf:4]: Actually not necessary... I removed that line since it didn't have any effect on the module.

**Review by mariomerco — APPROVED** [review]
LGTM!

**Pull request merged**